patterns/hero-product-split.php

```diff
@@ -6,17 +6,20 @@
  */
 ?>
 
-<!-- wp:media-text {"align":"full","mediaPosition":"right","mediaType":"image","mediaSizeSlug":"full","imageFill":false,"backgroundColor":"contrast","textColor":"base"} -->
-<div class="wp-block-media-text alignfull has-media-on-the-right is-stacked-on-mobile has-base-color has-contrast-background-color has-text-color has-background">
-	<div class="wp-block-media-text__content"><!-- wp:heading -->
-		<h2 class="wp-block-heading">Get cozy this fall with flannel shirts</h2>
+<!-- wp:media-text {"align":"full","mediaPosition":"right","mediaType":"image","mediaSizeSlug":"full","imageFill":false,"style":{"color":{"background":"#000000","text":"#ffffff"}}} -->
+<div class="wp-block-media-text alignfull has-media-on-the-right is-stacked-on-mobile has-text-color has-background" style="color:#ffffff;background-color:#000000">
+	<div class="wp-block-media-text__content">
+		<!-- wp:heading {"textColor":"background"} -->
+		<h2 class="wp-block-heading has-background-color has-text-color">Get cozy this fall with flannel shirts</h2>
```

Comment:
I know it's not introduced in this PR, but these strings should be localizable, in other words, they should be wrapped in `__()`, no?

Reply:
Agreed!

I've translated and escaped all rendered text in 2be2d29.